src/test/resources/risc-v/sw.riscv

```diff
@@ -2,6 +2,7 @@
   .align 2       # Make sure we're aligned to 4 bytes
   .globl _start
 _start:
+    addi t0, x0, 7
```

Comment:
Why this change?

Reply:
this was when I was testing store words in gem5 since I couldn't initialize the registers easily. I'll revert back now that I know stores work